src/adv.py

```diff
@@ -45,7 +49,7 @@
 
 player_name = input("\nWho are you? ")
 
-print(f'\nWelcome, {player_name}. An adventure awaits you\nshould you choose to accept it.\n')
+print(f'\n   {player_name}, you awaken to the faint scent of brimstone and lingering smoke. As you lie in the dirt you try to recall how you got to this place, but your mind is racing, and your thoughts escape you.\n\n   Where you are you do not know. However this place, it seems somewhat familiar... \n\n   You stand up and look around, the forest around you, or at least what remains is burnt and charred, all that lies in many places is ash. \n\n   You feel around in your pockets, nothing, nothing... There! Something in your left breast pocket, the one you never use.... A compass, hopefully it comes in handy.   \n\n   At this moment something unmistakeable catches your ear, a scream.\n\n   Not a horror movie theme park thriller scream either, this is bloodcurdling, throaty, desperate, a fight for your life kind of scream.\n\n   Your goose bumps have goose bumps, the hairs on your neck bristle so hard you feel you shirt lift up ever so slightly. \n\n  You take out your compass, the scream came from the west. \n\n  {player_name}, what do you do?  \n')
```

Comment:
There are some cool formatting solutions available for terminal output, check them out!